toDoList.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>To-Do List</title>
+        <link rel='stylesheet' href='https://use.fontawesome.com/releases/v5.0.6/css/all.css'>
+    </head>
+    <body>
+
```

Comment:
The only other thing I will say is that there are a lot of blank lines and it makes the page a little harder to read. After the opening body tag, for example, you don't really gain anything from the extra space. This is similar to the opening html and head tags - the tabbed in formatting makes it easy to read without having to include an extra space. Come talk to me if you're not sure what I mean, or do not agree